Extensions/FileSystem/JsExtension.js

```diff
@@ -385,6 +385,19 @@ module.exports = {
       .setIncludeFile('Extensions/FileSystem/filesystemtools.js')
       .setFunctionName('gdjs.fileSystem.getUserdataPath');
 
+    extension
+      .addStrExpression(
+        'UserDirPath',
+        _('User\'s Home folder'),
+        _('Get the path to user\'s folder. (On Windows C:\\Users\\<USERNAME>\\ for example)'),

```

Comment:
`Get the path to the user home folder (on Windows C:\\Users\\<USERNAME>\\ for example).`